### ISSUE TITLE
train with different learning_rate_constant, but got similar train & dev loss, also similar test bleu

### ISSUE THREAD

**Issue by Zminghua** [tensorflow/tensor2tensor#1799]
I download Arabic-Chinese parallel corpus from [opus](http://opus.nlpl.eu/) as train data and [xnli](https://github.com/facebookresearch/XNLI) as dev & test data, apply [sentencepiece](https://github.com/google/sentencepiece) to encode raw text into sentence pieces/ids, then train arabic-to-chinese translation model with learning_rate_constant=2 and learning_rate_constant=3, and keep other parameters default. However, I got similar results, including train loss, dev loss, and test bleu. Any idea or suggestion for this? Thanks!

![WechatIMG9](https://user-images.githubusercontent.com/14309435/76394788-fee47380-63b0-11ea-8210-4138d96661c5.jpeg)
![WechatIMG10](https://user-images.githubusercontent.com/14309435/76394807-086ddb80-63b1-11ea-95c0-85dc8b5e2c58.jpeg)
![WechatIMG11](https://user-images.githubusercontent.com/14309435/76394818-0dcb2600-63b1-11ea-8683-2a6a67bb0608.jpeg)
![WechatIMG12](https://user-images.githubusercontent.com/14309435/76394830-13287080-63b1-11ea-80f0-a82f09c6cf52.jpeg)
